api/handlers.go

```diff
@@ -575,9 +575,6 @@ func getContainersJSON(c *context, w http.ResponseWriter, r *http.Request) {
 		// make changes without messing with cluster.Container.
 		tmp := (*container).Container
 
-		// Update the Status. The one we have is stale from the last `docker ps` the engine sent.
-		// `Status()` will generate a new one
-		tmp.Status = cluster.FullStateString(container.Info.State)
```

Comment:
If `FullStateString` is no longer used , let's removed it from https://github.com/dani-docker/swarm/blob/1ca94a021a2495e274483d4be138da6413b9967b/cluster/container.go#L55

Also, please make sure the engine returns the same format state

Reply:
Yes, pushed a new commit for removing `FullStateString`.